desktop/logger.js

```diff
@@ -0,0 +1,13 @@
+import path from 'path';
+import { app } from 'electron';
+
+const logger = require('electron-log');
+
+logger.transports.file.file = path.join(app.getPath('userData'), '/app-logs.txt');
+
+export const writeInfo = (file, fn, res, args) =>
+  args
```

Comment:
prevent code repetition with single line:
logger.info(`In ${file}, in ${fn}, received: ${JSON.stringify(res)}, args ? called with arguments: ${JSON.stringify(args)}` : '')

Reply:
good catch, just made the fix!